pkg/processor/processor.go

```diff
@@ -109,6 +109,9 @@ func (p *Processor) buildPayload(apikeyString, redactedMessage, extraContent []b
 // and a copy of the message with some fields redacted, depending on config
 func (p *Processor) applyRedactingRules(msg message.Message) (bool, []byte) {
 	content := msg.Content()
+	if msg.IsTruncated() {
+		content = append([]byte(config.TruncWarningMsg), content...)
```

Comment:
So, we added a boolean to all the message constructors to be able to know we have to append the warning; Could this be done earlier, so we could leave the constructors intact ?